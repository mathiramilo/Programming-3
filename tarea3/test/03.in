# no siempre hay capacidad, sin reinicios
leer_datos
4 3 2 100 .
leer_rendimiento
4 3 2 1 .
max_datos_procesados
planificacion_optima
Fin