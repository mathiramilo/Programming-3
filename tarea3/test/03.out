1>2>3>4>Max Datos Procesados: 11
5>Cantidad Datos Planificacion: 11
6>Fin.
